### PR TITLE
BUGFIX: SD-2682 - Remove Web Component option from embed dialogs for Bear

### PR DESCRIPTION
**Pull request by nguyenson18296**
JIRA: SD-2682

---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12568/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14462/ : SUCCESS in 14m 08s

**Comment by nguyenson18296** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2016/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12570/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/373/ : UNSTABLE in 22m 36s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12574/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14466/ : SUCCESS in 12m 00s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12576/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14468/ : SUCCESS in 11m 57s

**Comment by nguyenson18296** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12577/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/374/ : FAILURE in 14m 43s

**Comment by nguyenson18296** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2018/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12579/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/375/ : SUCCESS in 22m 18s

File: libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/EmbedInsightDialogBase.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export type IEmbedInsightDialogBaseProps = {
     onCopyCode: (code: string, type: CopyCodeOriginType, codeType: EmbedType) => void;
     onOptionsChange: (opt: IReactOptions | IWebComponentsOptions) => void;
     onTabChange: (selectedTab: EmbedType) => void;
+    showWebComponentsTab?: boolean;
```
Comment (anvuong119) [inline libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/EmbedInsightDialogBase.tsx:29]: Please move this prop to line 23, as props which are non-function should be next to each other, for the sake of consistency.

File: libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx [hunk 2]
```diff
@@ -79,6 +80,30 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
                         action("onOptionsChange");
                     }}
                     openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={true}
```
Comment (anvuong119) [inline libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx:83]: Please move this prop to line 74, as props which are non-function should be next to each other, for the sake of consistency.

File: libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx [hunk 3]
```diff
@@ -79,6 +80,30 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
...
+                    integrationDocLink={"https://sdk.gooddata.com/gooddata-ui/docs/platform_integration.html"}
+                    onClose={action("onClose")}
+                    onCopyCode={action("onCopyCode")}
+                    onTabChange={action("onTabChange")}
+                    onOptionsChange={(opts) => {
+                        setReactOptions(opts as IReactOptions);
+                        action("onOptionChange");
+                    }}
+                    openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={false}
```
Comment (anvuong119) [inline libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx:106]: Please move this prop to line 97, as props which are non-function should be next to each other, for the sake of consistency.

File: libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx [hunk 4]
```diff
@@ -101,6 +126,7 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
                         action("onOptionChange");
                     }}
                     openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={true}
```
Comment (anvuong119) [inline libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx:129]: Please move this prop to line 120, as props which are non-function should be next to each other, for the sake of consistency.

File: libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx [hunk 5]
```diff
@@ -123,6 +149,7 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
...
```
Comment (anvuong119) [inline libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx:152]: Please move this prop to line 143, as props which are non-function should be next to each other, for the sake of consistency.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12626/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14515/ : SUCCESS in 11m 49s

File: common/changes/@gooddata/sdk-ui-all/SD-2682_2023-04-18-08-48.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "remove web components tab option from embed dialogs for bear",
```
Comment (anvuong119) [inline common/changes/@gooddata/sdk-ui-all/SD-2682_2023-04-18-08-48.json:5]: Please update the comment a little bit, something like below:

```
Make it possible to hide/show on demand the web components tab on embedding dialogs
```

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/12627/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 2m 13s
- gooddata-ui-sdk-unit-tests : SKIPPED

**Review by anvuong119 — APPROVED** [review]

**Comment by anvuong119** [comment]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3654

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2617/ : SUCCESS in 11m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/499/ : FAILURE in 1m 25s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/14516/ : SUCCESS in 11m 51s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/81/ : SUCCESS in 9m 12s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/117/ : SUCCESS in 11m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3773/ : SUCCESS in 13s

**Pull request merged**